x/logic/types/params.go

```diff
@@ -18,7 +18,7 @@ var (
 var (
 	DefaultRegisteredPredicates = make([]string, 0)
 	DefaultBootstrap            = ""
-	DefaultMaxGas               = math.NewUint(uint64(100))
+	DefaultMaxGas               = math.NewUint(uint64(10000))
```

Comment:
I'm wondering if it is sufficient, this is an open question, any remarks @ccamel ?

Reply:
@ccamel @bdeneux I propose to set a limit of `100000` for testing purposes. We'll easily find out what we're capable of with it, and it'll help identify a sufficient and reasonable limit for mainnet.

Reply:
```suggestion
	DefaultMaxGas               = math.NewUint(uint64(100000))
```
I'm agree with this value, I think it's a good start 👍